src/main/java/de/czymm/serversigns/translations/MessageHandler.java

```diff
@@ -151,7 +151,7 @@ private void copyEmbedded(String resourceName, Path target) {
                 Files.deleteIfExists(target);
                 Files.createFile(target);
 
-                try (BufferedWriter writer = Files.newBufferedWriter(target, Charset.defaultCharset()); BufferedReader reader = new BufferedReader(new InputStreamReader(resource))) {
+                try (BufferedWriter writer = Files.newBufferedWriter(target, Charset.forName('UTF-8')); BufferedReader reader = new BufferedReader(new InputStreamReader(resource))) {
```

Comment:
There is also a use of **defaultCharset** inside **ConfigGenerator** and **AdminListener**. Can you change them ?

Reply:
You can use `StandardCharsets.UTF_8` instead of `Charset.forName`

Reply:
done